### DISCUSSION TITLE
Official forum topics  - ownership problem

### DISCUSSION THREAD

**Discussion by kdecks** [RetroAchievements/RAWeb#661] [Ideas]
Official forum topics first post not editable by devs (if another user opened it).
It also makes it seem like the one who opened the topic "owns" it or is the lead developer.

Either 
1) make it so official forum threads topics are created by the **system** instead of users
and 
2) Developers need permission to edit this specific first post. (but not other posts)
or just 2.

3) What else? What could be a good solution here?

**Details:**
The problem is the first topic is used for providing the most important information about the game. Usually developers will need to this post add or make changes, and they only can if they've started the topic or they need to ask an admin to make changes.

**Comment by luchaos** [comment]
> The problem is the first topic is used for providing the most important information about the game. 

what kind of information is that usually? a lot of what happens in comments and forum posts is mostly a workaround to provide data that should be a default property of a game?

agreed on the system user creating the first post.

**Comment by luchaos** [comment]
Games will have dedicated forums instead of a single topic. 
Paired with topics that can be pinned, locked, and tagged this should not be an issue anymore.
